doc/users/dflt_style_changes.rst

```diff
@@ -418,100 +425,526 @@ or by setting::
+
+   classic_rcparams = {'image.interpolation': 'bilinear',
+                       'image.resample': False}
+
+   demo(ax1, classic_rcparams, 'classic')
+   demo(ax2, {}, 'v2.0')
+
+
+To restore the previous behavior set::
+
+   mpl.rcParams['image.interpolation'] = 'bilinear'
+   mpl.rcParams['image.resample'] = False
+
+or set::
+
+  image.interpolation  : bilinear  # see help(imshow) for options
+  image.resample  : False
+
+in your :file:`matplotlibrc` file.
+
+Colormapping pipeline
+---------------------
+
+Previously, the input data was normalized, then color mapped, and then
+resampled to the resolution required for the screen.  This meant that
+the final resampling was being done in color space.  Because the color
+maps are not generally linear in RGB space colors not in the color map
+may appear in the final image.  This bug was addressed by an almost
+complete overhaul of how the image handling code works.
+
+The input data is now normalized, then resampled to the correct
+resolution (in normalized dataspace), and then finally color mapped to
+RGB space.  This ensures only colors actually in the color map appear
+in the final image (if your viewer subsequently resamples the image
+the artifact may reappear).
+
+The previous behavior can not be restored.
+
+
+Shading
+-------
+
+- The default shading mode for light source shading, in
+  ``matplotlib.colors.LightSource.shade``, is now ``overlay``.
+  Formerly, it was ``hsv``.
+
+
 Plot layout
 ===========
 
-- The number of ticks on an axis is now automatically determined based
-  on the length of the axis.
+Ticks
+-----
 
-- The limits of an axes are scaled to exactly the dimensions of the data,
-  plus 5% padding.  The old behavior was to scale to the nearest "round"
-  numbers.  To use the old behavior, set the ``rcParam``
-  ``axes.autolimit_mode`` to ``round_numbers``.  To control the
-  margins on a particular side individually, pass any of the following
-  to any artist or plotting function:
+Direction
+~~~~~~~~~
+
+To reduce the collision of tick marks with data, the default ticks now
+point outward by default.  In addition, ticks are now only drawn on
+the bottom and left spines to reduce the plot looking like a
+porcupine.
+
+
+.. plot::
+
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+   import numpy as np
+
+   th = np.linspace(0, 2*np.pi, 128)
+   y = np.sin(th)
+
+   def demo(fig, rcparams, title, j):
+       np.random.seed(2)
+       with mpl.rc_context(rc=rcparams):
+
+           ax = fig.add_subplot(2, 2, j)
+           ax.hist(np.random.beta(0.5, 0.5, 10000), 25, normed=True)
+           ax.set_xlim([0, 1])
+           ax.set_title(title)
+
+           ax = fig.add_subplot(2, 2, j + 2)
+           ax.imshow(np.random.rand(5, 5))
+
+   classic = {'xtick.direction': 'in',
+              'ytick.direction': 'in',
+              'xtick.top': True,
+              'ytick.right': True
+   }
```

Comment:
Very tiny style nit: I think that the closing curly should be indented by the same amount as the opening curly; having it way over on the left is jarring and looks unpythonic to my eye.